pkg/cluster/internal/providers/podman/provider.go

```diff
@@ -182,20 +182,45 @@ func (p *Provider) GetAPIServerEndpoint(cluster string) (string, error) {
 		return "", errors.Errorf("network details should only be one line, got %d lines", len(lines))
 	}
 
-	// portMapping maps to the standard CNI portmapping capability
+	// portMapping19 maps to the standard CNI portmapping capability used in podman 1.9
```

Comment:
this also applies to other < 2.0, right?

Reply:
yes, this is the struct used by podman <= 1.9

Reply:
maybe we should note that /shrug